UI/src/model/selectedMassTransitStop.js

```diff
@@ -209,16 +245,22 @@
       });
     };
 
-    var requiredPropertiesMissing = function() {
-      var isRequiredProperty = function(publicId) {
+    var requiredPropertiesMissing = function () {
+      var isRequiredProperty = function (publicId) {
+        var isNormalBusStop = currentAsset.stopTypes && isTerminalType(currentAsset.stopTypes[0]) && _.some();
```

Comment:
is that _.some() ok ?
That will return false always, so the isNormalBusStop always be false

Reply:
Agree this function isRequiredProperty must have some tweaks. Check changes made again to see if everyone agrees

Reply:
For me it's ok.
Let's wait for the approval of @ricardofmsimoes since this was open by him.